### ISSUE TITLE
Language server fails to start up

### ISSUE THREAD

**Issue by Benjamin-van-Heerden** [zed-industries/zed#16109]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

After recent update, language server for Elixir no longer starting. Other language servers seem to work fine. I cannot even provide LSP logs since the LSP does not boot.  When inspecting the main Zed.log, I can see that there is an attempt to start the language server, but it fails due to `[ERROR] failed to start language server "elixir-ls": failed to spawn command. path: "/home/benjamin/.local/share/zed/extensions/work/elixir/elixir-ls-v0.23.0/language_server.sh", working directory: "/home/benjamin/Documents/Code/linkly", args: []`.

Tried manually deleting the LS in .../work/elixir and reinstalling, but still does not work.

Have attached the full related messages in Zed.log - those lines repeat multiple times.

### Environment

Zed: v0.147.2 (Zed)
OS: Linux X11 pop 22.04
Memory: 31.2 GiB
Architecture: x86_64
GPU: NVIDIA GeForce RTX 3050 || NVIDIA || 555.58.02

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>
Caused by:
    No such file or directory (os error 2)
2024-08-12T11:44:22.02057212+02:00 [INFO] starting language server "elixir-ls", path: "/home/benjamin/Documents/Code/linkly", id: 5
2024-08-12T11:44:22.334150618+02:00 [INFO] starting language server. binary path: "/home/benjamin/.local/share/zed/extensions/work/elixir/elixir-ls-v0.23.0/language_server.sh", working directory: "/home/benjamin/Documents/Code/linkly", args: []
2024-08-12T11:44:22.33496906+02:00 [ERROR] failed to start language server "elixir-ls": failed to spawn command. path: "/home/benjamin/.local/share/zed/extensions/work/elixir/elixir-ls-v0.23.0/language_server.sh", working directory: "/home/benjamin/Documents/Code/linkly", args: []
2024-08-12T11:44:22.33498389+02:00 [ERROR] server stderr: Some("")
2024-08-12T11:44:22.33499593+02:00 [INFO] retrying installation of language server "elixir-ls" in 1s
2024-08-12T11:44:23.335035055+02:00 [INFO] About to spawn test binary
2024-08-12T11:44:23.335052141+02:00 [WARN] test binary failed to launch
2024-08-12T11:44:23.335057968+02:00 [WARN] test binary check failed
2024-08-12T11:44:23.335063787+02:00 [INFO] beginning to reinstall server
2024-08-12T11:44:23.335074579+02:00 [INFO] deleting server container
2024-08-12T11:44:23.33521166+02:00 [ERROR] server container removal
</pre></details>

**Comment by arthurcogo** [comment]
I was getting this error message as well. Based upon this [comment](https://github.com/zed-industries/zed/issues/15802#issuecomment-2270085091) I tried making `language_server.sh` executable. Then the error message changed to be like the errors showing in https://github.com/zed-industries/zed/issues/15802. When I changed all the files to be executable, as recommended in that comment, things start working again :-D
